index.js

```diff
@@ -65,6 +65,11 @@ function flameGraph (opts) {
   var hoverFrame = null
   var currentAnimation = null
 
+  // Use custom coloring function if one has been passed in
+  if (opts.colorHash) colorHash = (d, decimalAdjust, allSamples, tiers) => {
+    return opts.colorHash(stackTop, { d, decimalAdjust, allSamples, tiers })
+  }
```

Comment:
Would you be so kind and document this option as well?

Reply:
Unless I've missed something, none of the d3-fg options are currently documented. 

They should be, though... I'll add a commit documenting them.